components/category/Category.js

```diff
@@ -58,10 +58,8 @@ export default function Category({
                 {filteredListItems.map((listItem, index) => (
                   <CategoryListItem
                     key={listItem.id}
-                    id={listItem.id}
+                    listItem={listItem}
                     index={index}
```

Comment:
why are you giving the CategoryListItem the index, if you already have an id? I don't know what you intend to with it, but I guess anything you want to do, you may also do with the id. it is also something that doesn't change if you delete items